test/parser/parser_spec.coffee

```diff
@@ -0,0 +1,12 @@
+chai = require 'chai'
+expect = chai.expect
+
+parse = require '../../parser/parser'
+outputTokens = require '../example/output_tokens'
+
+describe 'Scanner', ->
+
+  describe 'parsing valid teascript programs', ->
+    describe 'valid teascript program #1', ->
+      context "when test program1.tea's tokens
+               are passed through the scanner", ->
```

Comment:
I agree - I was having trouble figuring out some way to test this though since it felt like we needed to have everything at least stubbed out prior to beginning the tests (because everything just falls through all the way down to the bottom almost.